pages/docs/installation/node.mdx

```diff
@@ -7,11 +7,18 @@ pullQuote: "This guide will help you run the Grouparoo application on your local
 
 This guide will help you run the Grouparoo application on your local machine using Node.js.
 
-### Step 1: Install Runtime and Databases
+## Step 1: Install Runtime and Databases
 
-Grouparoo is a [node.js](https://nodejs.org/) application and requires Node.js version 12 or higher. You can download Node.js from their [website](https://nodejs.org/) or through the package manager for your operating system. The same is true for Postgres and Redis.
+Grouparoo is a [node.js](https://nodejs.org/) application and requires Node.js version 12 or higher. You can download Node.js from their [website](https://nodejs.org/) or through the package manager for your operating system.
 
-We also need to create a database within Postgres for Grouparoo to use. Let's use one called `grouparoo_development`.
+By default, running Grouparoo locally relies on the following dependencies:
```

Comment:
I think the default (ie: what `grouparoo init` sets up for you) is to use mock redis and sqlite.

Reply:
We should advise them to change these defaults when they are ready to persist data